tests/Controller/CRUDControllerTest.php

```diff
@@ -538,7 +488,7 @@ public function testConfigureWithException(): void
     public function testConfigureWithException2(): void
     {
         $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessage('Found service "nonexistent.admin" is not a valid admin service');
+        $this->expectExceptionMessage('You have requested a non-existent service "nonexistent.admin".');
```

Comment:
this actually shows the issue nicely.

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/Pool.php#L353

A non-existent service will result in this exception.